eox_tagging/models.py

```diff
@@ -308,14 +308,14 @@ def full_clean(self, exclude=None, validate_unique=False):
         self.clean()
         self.clean_fields()
 
-    def save(self, *args, **kwargs):  # pylint: disable=arguments-differ
+    def save(self, *args, **kwargs):  # pylint: disable=signature-differs
```

Comment:
same comment as above

Reply:
This one is because that's how they do it in the [Dango docs](https://docs.djangoproject.com/en/2.2/topics/db/models/#overriding-model-methods)